app/views/ops/_settings_server_tab.html.haml

```diff
@@ -214,7 +214,7 @@
         = _("Authentication")
       .col-md-8
         = select_tag('smtp_authentication',
-                      options_for_select(GenericMailer.authentication_modes, @edit[:new][:smtp][:authentication]),
+                      options_for_select(_(GenericMailer.authentication_modes), @edit[:new][:smtp][:authentication]),
```

Comment:
`GenericMailer.authentication_modes` is an array and you cannot feed an array to a gettext.